dashboard/app/helpers/aichat_helper.rb

```diff
@@ -0,0 +1,56 @@
+module AichatHelper
+  SAGEMAKER_CLIENT = Aws::SageMakerRuntime::Client.new
+  SAGEMAKER_MODEL_ENDPOINT = "mistral-7b-inst-v01" # "BioMistral-7B"
+  BASE_PROMPT = "You are a helpful chatbot for children. Limit your responses to a small paragraph."
+  ASSISTANT = 'assistant'
+  USER = 'user'
+
+  # The instruction-tuned version of Mistral accepts formatted instructions where conversation roles
+  # must start with a user prompt and alternate between user and assistant.
+  def self.format_inputs_for_sagemaker_request(aichat_params, stored_messages, new_message)
+    stored_messages.push({role: USER, content: new_message})
+    inputs = BASE_PROMPT + aichat_params[:systemPrompt]
```

Comment:
Does this need whitespace between?

Reply:
I don't believe so.

Reply:
Actually, I'll add since I see retrieval contexts are separated by whitespace as well. Thanks!